### PR TITLE
fixes and and feats, what can i say

### PR DESCRIPTION
**Pull request by Sv443**
- [x] Fix /steam PageEmbed timeout set to 10s
- [x] Move dev ids to .env
- [x] Add /whois command
- [x] Make /exec transpile TS code

### CODE REVIEW

File: src/Command.ts [hunk 1]
```diff
@@ -250,6 +250,8 @@ export abstract class Command
         {
             const embeds = [ embedify(`Couldn't run the command due to an error${err instanceof Error ? `: ${err.message}` : "."}`, settings.embedColors.error) ];
 
+            console.error(`Error while running the command ${k.yellow(`/${int.commandName}`)}\n`, err);
```
Comment (Sv443) [inline src/Command.ts:253]: idk why we didn't add this sooner, imo it's really useful

File: src/settings.ts [hunk 2]
```diff
@@ -45,27 +49,56 @@ export const settings: Settings = {
...
+export function getEnvVar<T extends ("string" | "number" | "stringArray" | "numberArray")>(varName: string, asType: T = "string" as T): undefined | (string | number | string[] | number[])
+{
+    const val = process.env[varName];
+
+    if(!val)
+        return undefined;
+
+    let transform: (value: string) => unknown = v => v.trim();
+
+    const commasRegex = /[.,،，٫٬]/g;
```
Comment (Sv443) [inline src/settings.ts:84]: (sometimes OSes like macos like to do a little trolling if you accidentally press modifier keys)

File: src/Command.ts (hunk 1 above)
Comment (IllusionMan1212) [reply src/Command.ts:253]: smh
https://github.com/codedrunks/BrewBot/pull/79/files#diff-dd56303679f34493449b477eeb5f9b1b85bc2c7f35227dee53267ce565601238

**Review by IllusionMan1212 — APPROVED** [review]
poggers and gude

File: src/Command.ts (hunk 1 above)
Comment (Sv443) [reply src/Command.ts:253]: wow I am schizophrenia

Comment (Sv443) [reply src/Command.ts:253]: mine looks better tho

**Pull request merged**